packages/web/components/main-menu.tsx

```diff
@@ -174,11 +178,11 @@ const MenuItemContent: React.FC<{
               {badge}
             </div>
             <div
-              className={`transition-visibility transition-opacity duration-300 ease-in-out ${
+              className={`transition-visibility mt-0 transition-opacity duration-300 ease-in-out ${
```

Comment:
There's an `mt-1` tag within this same string. IMO using `classNames` utility function instead of string interpolation helps make this a bit easier to avoid.